server/game/cards/04.1-FJ/Confession.js

```diff
@@ -0,0 +1,65 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class Confession extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Noon: Miracle 6',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            difficulty: 6,
+            target: {
+                activePromptTitle: 'Select a dude to add bounty to',
+                waitingPromptTitle: 'Waiting for opponent to select a dude',
+                cardCondition: { 
+                    location: 'play area', 
+                    condition: card =>
+                        card.gamelocation === this.gamelocation || 
+                        card.getGameLocation().isAdjacent(this.gamelocation)
+                },
+                cardType: ['dude'],
+                gameAction: 'addBounty'
+            },
+            onSuccess: (context) => {
+                this.game.resolveGameAction(GameActions.bootCard({ card: this.parent }), context);
```

Comment:
I know it is not strictly a cost, but since it has to be fulfilled, I would put the booting of parent to costs.

Reply:
I didn't put it as a cost, since it only happens if the spell pull is successful. Unless I misunderstand, if I put it as a cost, it will boot the parent even if the spell pull is unsuccessful. I can add something to the condition to make sure that the parent is not already booted though, to make sure that it can happen.

Reply:
You are right .. but there can be a cannot effect on the parent, so at least I would put the message in `thenExecute`

Reply:
Good point, will do.